src/main/java/bsh/org/objectweb/asm/Frame.java

```diff
@@ -240,7 +240,7 @@
    */
```

Comment:
NO! We don't make changes to package `bsh.org.objectweb.asm.*`

Reply:
Ooops. Agreed. Will revert.